lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -57,6 +57,21 @@ class SurfaceVertices implements ui.Vertices {
     }
     return list;
   }
+
+  bool _disposed = false;
+
+  @override
+  void dispose() {
+    _disposed = true;
+  }
+
+  @override
+  bool get debugDisposed {
+    if (assertionsEnabled) {
+      return _disposed;
+    }
+    throw StateError('Vertices.debugDisposed is only avialalbe when asserts are enabled.');
```

Comment:
Nit: Should this say SurfaceVertices?

Reply:
No, the user facing object is `Vertices`, which is abstract on web.